x-pack/plugins/serverless_search/public/application/constants.ts

```diff
@@ -12,5 +12,21 @@ export const INDEX_NAME_PLACEHOLDER = 'index_name';
 
 // Paths
 export const BASE_CONNECTORS_PATH = 'connectors';
+export const BASE_WEB_CRAWLERS_PATH = 'web_crawlers';
 export const EDIT_CONNECTOR_PATH = `${BASE_CONNECTORS_PATH}/:id`;
+export const ELASTIC_MANAGED_CONNECTOR_PATH = '/elastic_managed';
+export const ELASTIC_MANAGED_WEB_CRAWLERS_PATH = '/elastic_managed';
 export const FILE_UPLOAD_PATH = '/app/ml/filedatavisualizer';
+
+export const CRAWLER = {
+  github_repo: 'https://github.com/elastic/crawler',
+  docker_doc:
+    'https://github.com/elastic/crawler?tab=readme-ov-file#running-open-crawler-with-docker',
+};
+
+export const CONNECTORS = {
+  github_repo: 'https://github.com/elastic/connectors',
+  self_managed_docs:
```

Comment:
We should get this link from doc links instead and it shouldn't exist here at all IMO.

Reply:
Done